web/src/App.jsx

```
@@ -872,7 +880,7 @@ const MessageComposer = memo(({ user, db, selectedThread, lineInboxMode, activeL
           ) : "Send"}
         </button>
       </div>
-      {status && <div className="compose-status" role="status" aria-live="polite">{status}</div>}
+      {status && <div className={`compose-status ${getStatusClass(status)}`} role="status" aria-live="polite">{status}</div>}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)